source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -169,7 +169,8 @@
         this.items(_.map(data.Evaluations, function (e) {
             return new Evaluation(e);
         }));
-    }
+        this.period = data.Period;
```

Comment:
If you will not use it, I thik that it is better to revert the change.

If you will no revert, I suggest you to consider define the field as default in constructor and maybe using an observable.